config/wink.php

```diff
@@ -11,7 +11,7 @@
     | to anything you want.
     */
 
-    'database_connection' => env('WINK_DB_CONNECTION', 'wink'),
+    'database_connection' => env('DB_CONNECTION', 'wink'),
```

Comment:
@Lloople check this :)

Reply:
I was thinking of something like this with my comment

```
'database_connection' => env('WINK_DB_CONNECTION', env('DB_CONNECTION', 'wink')),
```

This way you can still use `WINK_DB_CONNECTION` but, if it's not defined, use the normal `DB_CONNECTION`

Reply:
It makes sense, I think it's a good idea what you're proposing. What do you think @francescomalatesta ??

Reply:
Why not? :) Let's see what @themsaid thinks about this.

Reply:
To me that makes more sense. It feels a bit awkward to fallback to a string. Maybe if we fallback to the ```DB_CONNECTION``` we could more explicit in the documentation to make sure the user understands it.